mypy/nodes.py

```diff
@@ -155,9 +155,6 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 class Expression(Node):
     """An expression node."""
-    literal = LITERAL_NO
-    literal_hash = None  # type: Key
```

Comment:
@ilevkivskyi isn't this a violation of strict optional? This value is actually read, it's not just a declaration.  I assume this should be `Optional[Key]`?
(This is why the build for this PR breaks; the new `literal_hash()` returns None when there's no match, which is what the client code expects).